rel/console.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+set -euo pipefail
+
+# This script connects to one of the running ECS tasks and opens an
+# interactive rails shell attached directly to the running system.
+# Idea stolen from https://github.com/thewca/wca-live/blob/main/rel/console.sh
+# You will need to have AWS CLI + Systems Manager Addon installed and credentials configured
+# in a profile called 'wca'
+
+# Colors for output
+COLOR_DEFAULT='\033[0m'
+COLOR_RED='\033[0;31m'
+COLOR_GREEN='\033[0;32m'
+
+export AWS_DEFAULT_PROFILE=wca
+
+usage() {
+  printf "${COLOR_DEFAULT}Usage: $0 -e <environment>\n"
+  printf " -e <environment>  Specify the environment (production or staging)"
+  exit 1
+}
+
+# Check if AWS CLI is installed
+printf "${COLOR_DEFAULT}AWS CLI | "
+command -v aws >/dev/null 2>&1 && status="$?" || status="$?"
+if [[ "${status}" = 0 ]]; then
```

Comment:
I believe you can do this:

```suggestion
if command -v aws >/dev/null 2>&1; then
```